.docker/Dockerfile-waf

```diff
@@ -1,73 +1,117 @@
-FROM owasp/modsecurity-crs@sha256:94c67e36f8c48553b4e51378579f6afc818a5d9d2e0c6e09cf846913ee9fac31
+ARG NGINX_VERSION="1.27.3"
+FROM nginx:${NGINX_VERSION}
 
 LABEL maintainer="USA.gov Web Ops"
 
-ARG RELEASE=3.3.2
-
 ARG GITBRANCH
-ENV GITBRANCH ${GITBRANCH:-none}
+ENV GITBRANCH=${GITBRANCH:-none}
 
 ARG GITCOMMIT
-ENV GITCOMMIT ${GITCOMMIT:-none}
+ENV GITCOMMIT=${GITCOMMIT:-none}
 
 ARG GITTAG
-ENV GITTAG ${GITTAG:-none}
+ENV GITTAG=${GITTAG:-none}
 
 ENV APP_NAME=USAGov \
-    PARANOIA=2 \
     ANOMALY_INBOUND=5 \
     ANOMALY_OUTBOUND=4 \
-    NGINX_KEEPALIVE_TIMEOUT=60s \
     ERRORLOG=/var/log/nginx/error.log \
     LOGLEVEL=warn \
-    USER=nginx \
-    PORT=80 \
-    SERVERNAME=locahost \
-    WORKER_CONNECTIONS=1024 \
     MODSEC_DEFAULT_PHASE1_ACTION="phase:1,pass,log,tag:'\${MODSEC_TAG}'" \
     MODSEC_DEFAULT_PHASE2_ACTION="phase:2,pass,log,tag:'\${MODSEC_TAG}'" \
-    MODSEC_RULE_ENGINE=on \
+    MODSEC_PCRE_MATCH_LIMIT=500000 \
+    MODSEC_PCRE_MATCH_LIMIT_RECURSION=500000 \
     MODSEC_REQ_BODY_ACCESS=on \
     MODSEC_REQ_BODY_LIMIT=13107200 \
     MODSEC_REQ_BODY_NOFILES_LIMIT=131072 \
     MODSEC_RESP_BODY_ACCESS=on \
     MODSEC_RESP_BODY_LIMIT=1048576 \
-    MODSEC_PCRE_MATCH_LIMIT=500000 \
-    MODSEC_PCRE_MATCH_LIMIT_RECURSION=500000
-
-COPY src-waf/docker-entrypoint-waf.sh /
-COPY src-waf/etc/ /etc/
-COPY src-waf/opt/ /opt/
-RUN rm -rf /opt/owasp-crs/tests;
+    MODSEC_RULE_ENGINE=on \
+    MODSECURITY_ENGINE_VERSION="3.0.13" \
+    MODSECURITY_NGINX_VERSION="1.0.3" \
+    NGINX_KEEPALIVE_TIMEOUT=60s \
+    PARANOIA=2 \
+    PORT=80 \
+    SERVERNAME=locahost \
```

Comment:
Typo (locahost -> localhost). 

I'm seeing some logging weirdness where it seems as if the waf is logging requests multiple times AND reporting 404s where it seems it shouldn't, and I wonder if this might somehow be the cause.

Reply:
Hmm, although I see we have always had this typo!

Reply:
still, fixed!